### PR TITLE
ci: Merge primitives-shared and primitives-event-dispatch groups

### PR DESCRIPTION
**Pull request by rahatarmanahmed**
This simplifies the approvals necessary for the event dispatch directory

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] ~~Tests for the changes have been added (for bug fixes / features)~~
- [ ] ~~Docs have been added / updated (for bug fixes / features)~~

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Previously PRs in the event-dispatch directory required reviews from both primitives-shared and primitives-event-dispatch groups. 

Issue Number: N/A

## What is the new behavior?

Both teams have agreed to merge, so now only one reviewer is necessary.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -849,26 +849,7 @@ groups:
         - mturco # Matt Turco
         - neonstalwart # Ben Hockey
         - nicholasyu-google # Nicholas Yu
-        - rahatarmanahmed # Rahat Ahmed
         - emspishak # Eric Spishak-Thomas
-    reviews:
-      required: 1
-      reviewed_for: optional
-    labels:
```
Comment (jatraman) [inline .pullapprove.yml:857]: Do we still need these labels?

Comment (rahatarmanahmed) [reply .pullapprove.yml]: Oops, fixed.

**Review by jatraman — APPROVED** [review]

**Review by thePunderWoman — CHANGES_REQUESTED** [review]
I am not in favor of this change. These groups were designed the way they were to ensure the people dedicated to the event-dispatch work had specific approvers and ownership of that area. The pullapprove group ensures one of them is requested for review on any PR affecting that section.

If we want to have them as part of the shared primitives group, too, I think that's fine, but we should leave both.

**Review by thePunderWoman — APPROVED** [review]
I'm ok with this due to offline conversations. LGTM

**Comment by atscott** [comment]
This PR was merged into the repository by commit 92debf44761603b0e8b8c9c8efbee17f25391c06.

**Pull request closed without merging**